src/test/java/seedu/address/testutil/DoctorBuilder.java

```diff
@@ -9,12 +9,13 @@
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Phone;
 import seedu.address.model.person.ShiftDays;
+import seedu.address.model.person.Specialisation;
 import seedu.address.model.util.SampleDataUtil;
 
 /**
  * A utility class to help with building Doctor objects.
  */
-public class DoctorBuilder {
+public class DoctorBuilder extends StaffBuilder {
```

Comment:
May I know if there are any implications in extending here? Not a problem at all, was just wondering if I have messed up by not extending previously.

Reply:
The intension is just to override build() in StaffBuilder iirc, probably optional